### ISSUE TITLE
Question about the implement  of the KLDiv Loss

### ISSUE THREAD

**Issue by appleleaves** [antspy/quantized_distillation#7]
As I can see in the file **help_fun.py**, you use **SoftmaxFunction** to normalize the teacher output but use **logSoftmaxFunction** to normalize the student output. 
Why do you set this two different?

**Comment by appleleaves** [comment]
Mates have the same question please refer to the pytorch implement of the KLDivLoss
